packages/google_maps_flutter/google_maps_flutter_web/example/integration_test/google_maps_controller_test.dart

```diff
@@ -568,6 +599,33 @@ void main() {
         }));
       });
 
+      testWidgets('updateHeatmaps', (WidgetTester tester) async {
+        final mock = MockHeatmapsController();
```

Comment:
We enabled some extra lints in this package, and this is definitely going to complain.

There's plenty of examples in the new code to follow, but you'll get a lot of "must specify types" below.

Reply:
Fixed